src/Equations/elastic/Kernels/Local.cpp

```diff
@@ -42,7 +42,6 @@
 #include "Kernels/Local.h"
 
 #ifndef NDEBUG
-#pragma message "compiling local kernel with assertions"
 #endif
```

Comment:
I guess `#ifndef` and `#endif` can be also removed, right?

Reply:
Yup, forgot that one